backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java

```diff
@@ -124,11 +127,20 @@ private static void migrateProjects() {
           for (ProjectEntity project : projectEntities) {
             boolean migrated = false;
             if (project.getOwner() != null && !project.getOwner().isEmpty()) {
-              WorkspaceDTO workspaceDTO =
-                  roleService.getWorkspaceDTOByWorkspaceId(
-                      userInfoMap.get(project.getOwner()),
-                      project.getWorkspace(),
-                      project.getWorkspace_type());
+              WorkspaceDTO workspaceDTO;
+              try {
+                workspaceDTO =
+                    roleService.getWorkspaceDTOByWorkspaceId(
+                        userInfoMap.get(project.getOwner()),
+                        project.getWorkspace(),
+                        project.getWorkspace_type());
+              } catch (StatusRuntimeException ex) {
+                if (ex.getStatus().getCode() == Status.Code.NOT_FOUND) {
+                  LOGGER.warn("Failed to get workspace (skipping) : " + ex.toString());
+                  continue;
```

Comment:
Need to think if we should mark this as migrated to avoid trying to migrate all over again.

Reply:
We should probably tag this as deleted.